signing/signing.go

```diff
@@ -0,0 +1,44 @@
+package signing
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/ossf/scorecard/v2/cron/data"
+)
+
+type ScorecardOutput struct {
+	SarifOutput string
+	JsonOutput  string
+}
+
+func GetResults(w http.ResponseWriter, r *http.Request) {
+	// Get params to build GCS filepath.
+	ctx := context.Background()
+	bucketURL := "gs://ossf-scorecard-results"
+	host := mux.Vars(r)["host"]
+	orgName := mux.Vars(r)["orgName"]
+	repoName := mux.Vars(r)["repoName"]
+	filePath := fmt.Sprintf("%s/%s/%s/results.json", host, orgName, repoName)
+
+	// Sanitize input and log query.
+	escapedFilePath := strings.Replace(filePath, "\n", "", -1)
+	escapedFilePath = strings.Replace(escapedFilePath, "\r", "", -1)
+	log.Printf("Querying GCS bucket for: %s", escapedFilePath)
```

Comment:
This bit of code is scary. We should do more validation to check for malicious input. I would suggest starting with `filepath.Clean` to sanitize the path and then use `filepath.Match("*/*/*/results.json")` to ensure that exactly 3 separators exist.

@laurentsimon @asraa in case you have better suggestions.